pkg/app/run.go

```diff
@@ -39,6 +39,19 @@ func (r *Run) askForConfirmation(msg string) bool {
 	return AskForConfirmation(msg)
 }
 
+func (r *Run) prepareChartsIfNeeded(helmfileCommand string, dir string, concurrency int, opts state.ChartPrepareOptions) (map[state.PrepareChartKey]string, error) {
```

Comment:
concurrency? it seem like not be handled?

Reply:
Now, wherever this function is called, valid concurrency values should be (greater than 0) are always provided. this is handled now